scripts/update/05-checkreqs.sh

```diff
@@ -20,7 +20,7 @@ if ! command -v echo_error > /dev/null 2>&1; then
     killage "Echo functions unavailable"
 fi
 
-if [[ -z $log ]]; then
+if [[ -z ${LOG} ]]; then
```

Comment:
Oh yeah, this change will probably trip up the updater. I'll make sure to add a switch from `$log` to `$LOG` there so that it won't interrupt an update.